awsbatch-cli/tox.ini

```diff
@@ -0,0 +1,293 @@
+[tox]
```

Comment:
I  think you need to update the GitHub CI workflow

Reply:
also don't we need to change anything in the cli tox file?

Reply:
done, thanks!